data/openldap_to_389ds/sssd.conf

```diff
@@ -0,0 +1,25 @@
+[sssd]
+config_file_version = 2
+domains = ldapdom
+services = nss, pam
+debug = 9
+
+[nss]
+filter_groups = root
+filter_users = root
+debug = 9
+
+[domain/ldapdom]
+debug = 9
+id_provider = ldap
+auth_provider = none
+
+ldap_schema = rfc2307bis
+ldap_uri = ldap://127.0.0.1
+ldap_tls_cacert = /tmp/ldap-sssdtest.cacrt
```

Comment:
Is this ca file used during your test? 
I noticed that ldap_uri = ldap (rather than ldaps)

Reply:
Not enable security currently.

Reply:
Then we can remove this line safely